crates/subspace-farmer/src/single_disk_plot.rs

```diff
@@ -329,6 +329,16 @@ pub enum SingleDiskPlotError {
     /// Node RPC error
     #[error("Node RPC error: {0}")]
     NodeRpcError(Box<dyn std::error::Error + Send + Sync + 'static>),
+    /// Insufficient allocated space
```

Comment:
Something like "Allocated space smaller than one sector" or similar that would be easier to read. This is too verbose. Also strings are not good here, it should probably be just bytes, this is a developer-facing API after all.